integration-tests/tests/test_utils/trait_mod/connection_router_trait.rs

```diff
@@ -0,0 +1,45 @@
+use cosmwasm_std::Addr;
```

Comment:
I'd prefer a different naming convention for these files. How about just calling it connection_router_contract.rs? And can we just put them all under src, next to the contract.rs file?

Reply:
ok sure.